actions/add-review-url/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "AddReviewUrl",
+  "version": "1.0.0",
+  "description": "",
+  "main": "AddReviewUrl.js",
+  "scripts": {
+    "build": "babel src -d lib",
+    "buildtest": "babel test -d spec",
+    "test": "jasmine spec/addReviewUrlTest.js"
+  },
+  "keywords": [],
+  "author": "s-tuli",
+  "license": "ISC",

```

Comment:
The license should match the license for the repo as a whole (MIT).

Reply:
Updated. In fact, I removed reference to this license in other package.json files as well.